src/addremove.js

```diff
@@ -0,0 +1,84 @@
+import { setToLocalStorage, getFromLocalStorage, reloadToDo } from './storage.js';
+import { refreshStore } from './functions';
+
+const editToDo = () => {
+  const toDoList = document.getElementsByClassName('task');
+  for (let i = 0; i < toDoList.length; i += 1) {
+    const taskLabel = toDoList[i].children[0].children[1];
+    taskLabel.addEventListener('change', () => {
+      refreshStore();
+    });
+  }
+};
+
+const addToDo = () => {
+  const buttons = document.getElementsByClassName('delete');
+  for (let i = 0; i < buttons.length; i += 1) {
+    buttons[i].addEventListener('click', () => {
+      const index = `item-${i}`;
+      const inputItem = document.getElementsByName(index)[0];
+      inputItem.parentElement.parentElement.remove();
+      refreshStore();
+      window.location.reload();
+    });
+  }
+};
+
+const appendToDo = (task) => {
+  const taskList = document.createElement('example');
+  taskList.innerHTML = `
+          <div class="task">
```

Comment:
Expected indentation for two fewer spaces ( indentation ) to make readable code.